dll/win32/advapi32/reg/reg.c

```diff
@@ -4847,10 +4847,7 @@ RegSetValueExA(HKEY hKey,
     /* Convert SubKey name to Unicode */
     if (lpValueName != NULL && lpValueName[0] != '\0')
     {
-        BOOL bConverted;
-        bConverted = RtlCreateUnicodeStringFromAsciiz(&ValueName,
-                                                  (PSTR)lpValueName);
-        if(!bConverted)
+        if (!RtlCreateUnicodeStringFromAsciiz(&ValueName, (PSTR)lpValueName))
```

Comment:
Is casting needed?

Reply:
It looks like not, but that is separate from this PR: I will fix the whole file in a follow-up PR.